synapse/tests/test_telepath.py

```diff
@@ -78,6 +78,11 @@ def test_telepath_basics(self):
         self.raises(SynErr, foo.baz, 10, 20)
 
         foo.fini()
+        # We have fini'd the Proxy resources
+        self.true(foo._tele_boss.isfini)
+        self.true(foo._tele_sock.isfini)
+        self.true(foo._tele_plex.isfini)
```

Comment:
This PR looks good to me. I ran the tests for this branch locally and they passed. I ran them again w/ the onfini commented out and they failed, demonstrating that the multiplexer would be left unfinid.